### PR TITLE
Update wording in various places

### PR DESCRIPTION
**Pull request by triceo**

### CODE REVIEW

File: docs/src/antora.yml
Comment (rsynek) [inline docs/src/antora.yml:1]: Can we still generate the docs without this file?

File: docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc [hunk 1]
```diff
@@ -558,16 +546,11 @@ There are additional parameters you can supply to your `solverConfig.xml`:
 <solver xmlns="https://timefold.ai/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://timefold.ai/xsd/solver https://timefold.ai/xsd/solver/solver.xsd">
   <moveThreadCount>4</moveThreadCount>
-  <moveThreadBufferSize>10</moveThreadBufferSize>
```
Comment (rsynek) [inline docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc:561]: Yep, this parameter has been always "a bit too much". Can we remove/deprecate it?

File: docs/src/antora.yml
Comment (triceo) [reply docs/src/antora.yml:1]: Yes. It is added automatically to the release branch by the CI.

File: docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc (hunk 1 above)
Comment (triceo) [reply docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc:561]: I'm open to the conversation. Feel free to start it.

**Review by rsynek — APPROVED** [review]

**Pull request merged**